install/api-manager/k8s-artifacts/wso2apim-deployment.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2apim
+  namespace: wso2
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      deployment: wso2apim
+  minReadySeconds: 30
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        deployment: wso2apim
+    spec:
+      containers:
+        - name: wso2apim
+          image: pubudu/wso2am:3.0.0-snapshot
```

Comment:
Don't we need to take this image to a common repo ?

Reply:
let's use this for the moment until we come up with the official v3 image.